packages/core/.npmignore

```diff
@@ -0,0 +1,15 @@
+# The paths in this file are specified so that they align with the file structure in `./build` after this file is copied
+# into it by the prepack script `scripts/prepack.ts`.
+
+*
+
+!/cjs/**/*
+!/esm/**/*
+!/types/**/*
+
+# These paths are necessary for Node AWS Lambda creation
+# This package is a (transitive) dependency of @sentry/serverless and thus it is pulled into
+# the lambda layer zip file. By specifying these paths, we
```

Comment:
I feel like there is a part of this sentence missing. Same applies to other files.

Reply:
Whoops, good catch - thanks for bringing it up! Fixed it.